stdlib/REPL/src/LineEdit.jl

```diff
@@ -65,7 +65,6 @@ show(io::IO, x::Prompt) = show(io, string("Prompt(\"", prompt_string(x.prompt),
 
 mutable struct MIState
     interface::ModalInterface
-    active_module::Module
```

Comment:
I don't know if we should delete this, as some REPL instances might not want to use the global `active_module`. Or could be nullable?